src/new-components/password-field/password-field.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react'
+import { useForkRef } from 'reakit-utils'
```

Comment:
The implementation of `useForkRef` looks almost identical to `useCombinedRef` we have within TD/TW. The project's [readme](https://github.com/reakit/reakit/blob/master/packages/reakit-utils/README.md) also warns that there may be breaking changes in minor releases, so to better prevent breakage we can either:

- Explicitly pin the current version of `reakit-utils` to package.json. I noticed right now we're using whichever version comes with reakit
- Port over `useCombinedRefs` as a utility file here in Reactist (and to replace the copy in TD/TW)

Reply:
I checked and we're already pinning down the reakit version to a specific version, which should make the reakit-utils stable, right?

I do want to host some of our reusable hooks in Reactist, but that's out of the scope for now.